congratulations/Congratulations.py

```diff
@@ -0,0 +1,52 @@
+from PIL import ImageTk, Image
+import webbrowser as webbrowser
+from urllib.parse import quote
+try:
+    import Tkinter as Tk        # Python < 3.0
+except ImportError:
+    import tkinter as Tk        # Python >= 3.0
+
+class Congratulations(object):
+    """Represents the completion of the goal. Houses functionality
```

Comment:
When using docstrings, it is recommended to have a oneline summary, and if further explanation is needed this should be done following a white space line. The first line (the summary) should start with a capital letter, and end with a period. This format is required to use documentation tools, so it would be neat to keep all functions this way.

Reply:
Good call! Will fix that up.